tests/unit/test_noise_form.py

```diff
@@ -147,19 +148,19 @@ def test_noise_order(mocker, dummy_data, dummy_config_noise_numbers):
                 field,
             )
 
+    # Get config for dataset
+    dummy_config = get_dummy_config_noise_numbers(dataset)
     # FIXME: would be better to mock the dataset instead of using census
-    noise_dataset(DATASETS.census, dummy_data, dummy_config_noise_numbers, 0)
+    noise_dataset(dataset, dummy_data, dummy_config, 0)
 
     # This is getting the string of each noise type. There are two mock calls
     # being made to each noise type with how we are mocking noise type attirbutes
     # above causing duplicates in the call list. Call order is each instance a noise
     # function is called. Here we grab the string of the noise type for one mock method
     # call and no the second mthod.
     call_order = [x[0] for x in mock.mock_calls if type(x[1][0]) == str]
-    expected_call_order = [
-        # NOISE_TYPES.omit_row.name,   # Census doesn't use omit_row
-        NOISE_TYPES.do_not_respond.name,
-        # NOISE_TYPES.duplicate_row.name,
+    row_order = [row_noise_type.name for row_noise_type in dataset.row_noise_types]
+    column_order = [
```

Comment:
Would it be useful to add a comment saying the provenance of the expected column order? (e.g., a link to the docs?)

Reply:
Are you saying more than the comment at the beginning of the test?

Reply:
No, I meant right by the definition of the expected ordering. It came from somewhere, if future devs want to know the provenance, what would they do?